docs/milestones.md

```diff
@@ -1,99 +0,0 @@
-
-__Approval__. REST endpoint to which the platform "asks permission" before running any job. The platform sends the approval endpoint information including: 
-
-1. Input and output storage 
-2. Which compute the job wishes to run in 
-3. The author of the code the job is running 
-4. Whether or not the job has been code-signed by the configured policies 
-
-The approval endpoint can either approve / reject the job based on checked-in configuration (e.g., of which storage accounts are associated with which silo) or pass this information on for manual approval. 
-
-:exclamation: Note that the approval endpoints do not support 3P-facing AML yet. 
-
-__Silo__. Isolated collection of storage and compute. Here, "isolated" means that the platform guarantees: 
-
-- Only compute within the silo can "touch" storage within the silo. 
-- Only data of intermediate or eyes-on classification can be moved outside the silo. 
-- Only "approved" jobs can change the classification of data or move it outside the silo. 
-
-Silos are expected to be reliable (i.e., no concerns around network connectivity or uptime). 
-
-:exclamation:  Note that we assume a hard cap of ≤ 100 silos at current stage. 
-
-__Orchestrator__. Collection of storage and compute. The storage is for model parameters, rather than the actual data. A task orchestrator broadcasts the FL task, sends the current model to each silo, and aggregates the gradients from the silos. In this file, orchestrator is assumed to live in an AML workspace. 
-
-
-## Key Milestones
-
-- A machine learning model. A working model or model architecture with learning algorithm in a non-federated scheme. 
-- Resource provisioning. Get the orchestrator and silos ready for FL. 
-- A test FL job. Run a synthetic test for the FL framework and understand the effect of each FL hyperparameters. 
-- An actual FL job. Set the FL hyperparameters and execute the actual job. 
-
-We describe the key milestones in detail below. 
-
-
-## Machine learning model 
-
-The prerequisite of a _federated_ learning job is a _machine_ learning model, which can be trained in a non-federated scheme. Such a model can be  
-- a working model trained with some real yet small data, or  
-- a model architecture that is proven effective via synthetic training data. 
-
-One example of a hospital (i.e., the customer) is described below. The hospital’s task is to classify a medical image into positive or negative of a specific disease. The data scientists at the hospital _already_ have a neural network model trained in a centralized fashion with their own patient data. The model achieved reasonable performance. Now the hospital wants to further improve the model's performance as more hospitals would like to participate in a federated learning way without their data leaving their clusters. 
```

Comment:
keep it for a page explaining why FL is good for you?

Reply:
Added it here https://github.com/Azure-Samples/azure-ml-federated-learning/pull/72/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R36